src/Forms/TextCheckboxGroupField.php

```diff
@@ -32,6 +32,58 @@ public function __construct($title = null)
         $this->setTitle($title);
     }
 
+    /**
```

Comment:
TextCheckboxGroupField is the special elemental field that is a composite field of the default `Title` and `ShowTitle` fields

Reply:
I've raised a new card for the refactor to do this. I feel like there's already a lot going on in this card and there's [another issue](https://github.com/silverstripe/silverstripe-elemental/issues/1170) that's dependent on the PRs in this issue

Reply:
If we keep it in this PR, that means you need to add unit tests here and then to refactor we're removing both the src code and the tests, which shouldn't have been added to this repo in the first place.
Moving them now doesn't introduce any new complexity, it just puts the new code in the right place the first time around.

Reply:
Like I said, there's already an [existing framework PR ](https://github.com/silverstripe/silverstripe-framework/pull/11213) that deals with RequiredFields that's required for this PR to work.  I don't want to add CompositeFields functionality to that PR because that has nothing to do with RequireFields, so I want to do a seperate PR.  I don't want to block this PR on another framework PR.

It's just easier to refactor this later I don't see the problem

Reply:
Actually nevermind - after refinment today I'm okay with this being done separately.

Reply:
[Other card](https://github.com/silverstripe/silverstripe-framework/issues/11215) has now been refined so we'll do on that card